src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -154,6 +154,23 @@ const debounceFrame = (func: DebouncableFn): DebouncedFn => {
 	};
 };
 
+const getIconElements = (): {[key: string]: ReactElement<any>} => {
+	const iconList = ['vm', 'cloud', 'tag', 'inbox', 'check', 'chrome-minimize'];
+	const elementLibrary: {[key: string]: ReactElement<any>} = {};
+	iconList.forEach(iconKey => {
+		elementLibrary[iconKey] = createElement('span', {className: `codicon codicon-${iconKey}`});
+	});
+	return elementLibrary;
+};
+
+const getIconElementLibrary = (): (key: string, props?: {[key: string]: string}) => ReactElement => {
+	const iconElementLibrary = getIconElements();
```

Comment:
Every time the callback is used in the library, we are creating again all the icons. This could degrade the performance of the graph.

My suggestion is to use a constant instead (with all the icons) and then use them here.